docs/BUILD.md

```diff
@@ -0,0 +1,13 @@
+1. Compile relevant binaries by running the following script:
+
+```sh
+./install_dependencies.sh
+```
+
+> This step can take ~15 minutes depending on your machine.
+
+2. Build Sneeffer
+
+```
+go build -o sniffer .
+```
```

Comment:
[Question]: Where is the run? 
[Statement]: We will complete the README after we complete the initial commit. No need to spend more time here.

Reply:
done